FLIR/conservator/util.py

```diff
@@ -53,3 +55,21 @@ def base_convert(b, n):
         output.append(r)
         n = int(n / b)
     return output
+
+
+def chunks(list, size):
+    """
```

Comment:
@rlc2 Is this format correct?

Reply:
It's closer, just needs a short description at the top of the docstring followed by a blank line to separate it from the :param lines.

Reply:
Oh also the note starting with "once the input list is exhausted" should be between the short description and the :param lines.  So to summarize:
"""
short description

additional comments

:param X: ...
"""